core/db/models/project_state.py

```diff
@@ -249,6 +249,7 @@ def complete_epic(self):
 
         log.debug(f"Completing epic {self.unfinished_epics[0]['name']}")
         self.unfinished_epics[0]["completed"] = True
+        self.tasks = []
```

Comment:
Hmmm I was thinking that we might save tasks (so I started adding epic_id to each of them so we can easily find them). I think we might need this info for later prompts. 

If this is just to support sending the list of tasks to vscode extension just for current extension, we should be able to filter them on epic_id (which may not exist as in theory we've talked about tasks not belonging to any epic in some special cases). So the decision to clear or not should be based on whether we'd need them in future prompts (and we always want to avoid reading previous states from pythagora).